fileio.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
```

Comment:
Few things: 
1. you didn't print each number on a newline
2. you didn't square them!